osu.Game/Overlays/Profile/Sections/PaginatedContainerHeader.cs

```diff
@@ -0,0 +1,132 @@
+{
+    public class PaginatedContainerHeader : CompositeDrawable, IHasCurrentValue<int>
+    {
+        public Bindable<int> Current
+        {
+            get => current;
+            set
+            {
+                if (value == null)
+                    throw new ArgumentNullException(nameof(value));
+
+                current.UnbindBindings();
+                current.BindTo(value);
+            }
+        }
+
+        private readonly Bindable<int> current = new Bindable<int>();
+
+        private readonly string text;
+        private readonly CounterVisibilityState counterState;
+
+        private CounterPill counterPill;
+
+        public PaginatedContainerHeader(string text, CounterVisibilityState counterState)
+        {
+            this.text = text;
+            this.counterState = counterState;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OverlayColourProvider colourProvider)
+        {
+            AutoSizeAxes = Axes.Both;
+            Padding = new MarginPadding { Vertical = 10 };
+            InternalChildren = new Drawable[]
+            {
+                new CircularContainer
+                {
+                    RelativeSizeAxes = Axes.Y,
+                    Height = 0.65f,
+                    Width = 3,
+                    Masking = true,
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreRight,
+                    Margin = new MarginPadding { Right = 10 },
+                    Child = new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Colour = colourProvider.Highlight1
+                    }
+                },
+                new FillFlowContainer
+                {
+                    AutoSizeAxes = Axes.Both,
+                    Direction = FillDirection.Horizontal,
+                    Spacing = new Vector2(10, 0),
+                    Children = new Drawable[]
+                    {
+                        new OsuSpriteText
+                        {
+                            Anchor = Anchor.CentreLeft,
+                            Origin = Anchor.CentreLeft,
+                            Text = text,
+                            Font = OsuFont.GetFont(size: 14, weight: FontWeight.Bold),
+                        },
+                        counterPill = new CounterPill
+                        {
+                            Anchor = Anchor.CentreLeft,
+                            Origin = Anchor.CentreLeft,
+                            Alpha = getInitialCounterAlpha(),
+                            Current = { BindTarget = current }
+                        }
+                    }
+                }
+            };
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            current.BindValueChanged(onCurrentChanged);
+        }
+
+        private float getInitialCounterAlpha()
```

Comment:
Why do it this way?

How about changing to `BindValueChanged(onCurrentChanged, true)` above, and then combining this into `onCurrentChanged`? The call to this can then be removed from `load()`.